include/gz/common/EnumIface.hh

```diff
@@ -142,6 +142,7 @@ namespace ignition
     ///   std::cout << "Type++ Name[" << myTypeIface.Str(*i) << "]\n";
     /// }
     /// \verbatim
+    GZ_UTILS_WARN_IGNORE__DEPRECATED_DECLARATION
```

Comment:
You'll need to include https://github.com/gazebosim/gz-utils/blob/ign-utils1/include/gz/utils/SuppressWarning.hh and use the `IGN` prefix. Unfortunately, these macros don't have a `GZ` prefixed alias in `ign-utils1`.